tooling/minimize/intrinsics/Cargo.toml

```diff
@@ -7,3 +7,6 @@ edition = "2021"
 
 [lib]
 crate-type = ["staticlib", "lib"]
+
+[workspace]
+members = ["."]
```

Comment:
What's up with this diff?

If the goal is to add this to the workspace, then that's not what you did. You made this a new workspace instead I think. Add it in `tooling/Cargo.toml` instead, please.

Reply:
Required for the rust.sh script in minimizer/tests to work, but will be implemented in a seperate PR.